src/li_chao.rs

```diff
@@ -0,0 +1,99 @@
+/// A structure for answering minimum queries on a set of linear functions. Supports two
```

Comment:
note: our other convex hull trick now searches for the max

Reply:
OK, I will flip everything to keep things consistent.

Reply:
Thanks